scripts/self_update.ts

```diff
@@ -54,3 +83,13 @@ async function update() {
 
   await (changedToMain ? runPullOriginMainCmd.status().then(() => printLogo()) : Promise.reject(textDecoder(error)));
 }
+
+async function storeLastCheckUpdateDate() {
+  try {
```

Comment:
As we've got the try-catch inside the `shouldCheckForUpdates` function, we might don't need this one. What do you think? Is there any flow that can trigger the catch here?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix